packages/app/src/providers/api.ts

```diff
@@ -55,26 +72,41 @@ export abstract class ProviderApi {
   protected async cancellableFetchValue<T>(
     endpoint: string,
     storeParams: ValuesStoreParams,
-    queryParams?: Record<string, string | boolean | undefined>,
+    queryParams: Record<string, string | boolean | undefined>,
     responseType?: ResponseType
```

Comment:
Note a minor change here: `queryParams` defaulting to `storeParams` was prone to bugs in my opinion. We no longer relied on it anyway.